scripts/test-template.sh

```
@@ -89,36 +106,102 @@ log_success "Test app created successfully"
+        log_error "Failed to install dependencies"
+        exit 1
+    fi
+
+    log_info "Checking @pikku package installation..."
+    for pkg in schedule modelcontextprotocol; do
+        pkg_dir=$(find node_modules/@pikku -maxdepth 1 -name "$pkg" -type d -o -name "$pkg" -type l 2>/dev/null | head -1)
+        if [ -n "$pkg_dir" ]; then
+            if [ -f "$pkg_dir/dist/index.d.ts" ]; then
+                log_success "@pikku/$pkg: dist/index.d.ts exists"
+            else
+                log_warning "@pikku/$pkg: dist/index.d.ts MISSING (ls: $(ls $pkg_dir/ 2>/dev/null))"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)